packages/frontend/apps/electron/scripts/dev.ts

```diff
@@ -40,7 +40,7 @@ function spawnOrReloadElectron() {
     process.env.NODE_OPTIONS = NODE_OPTIONS.replace(/--import=[^\s]*/, '');
   }
 
-  spawnProcess = spawn(exe, ['.'], {
+  spawnProcess = spawn(exe, ['.', '--inspect'], {
```

Comment:
will it slow down the startup time?

Reply:
In my experience it won't. Also this only affects dev